steamcmd-install.sh

```diff
@@ -40,6 +41,14 @@ if [[ "$?" != 0 ]]; then
 fi
 #TODO : implement a md5 check
 su $user -c "cd $steamcmd && tar -xvzf steamcmd_linux.tar.gz"
+su $user -c chkhash=$(md5sum steamcmd_linux.tar.gz | cut -d' ' -f1)
+if test "$chkhash" == "09e3f75c1ab5a501945c8c8b10c7f50e" 
```

Comment:
In my opinion, the hash should be a variable
And, it could be usefull to implement a --force option (but that could be later)